locales/pt_BR.json

```diff
@@ -0,0 +1,121 @@
+          "sodiumiris": "Vanilla + <span class='text-span-4'>Sodium</span> + <span class='text-span-3'>Iris</span>",
+		  "sodiumirismessage": "Iris quando usado em conjunto com Minecraft Vanilla e Sodium entrega algo que nenhum outro mod do fabric jamais fez: Dar suporte a shaders. Você pode usar shaders com um framerate surpreendentemente alto (comparado ao OptiFine) sem precisar recorrer para upgrades em seu computador ou gambiarras no jogo Vanilla. E apesar do Iris custar entre 10 a 15% dos frames, graças ao Sodium os FPS continuam similares ou até mesmo maiores que o jogo Vanilla. No futuro a Iris estará totalmente integrada e otimizada, tendo até mesmo as próprias otimizações gráficas (assim como OptiFine), fazendo com que ambos Sodium e Iris se agregem ao ponto do desempenho deles somados seja maior que usar apenas um. E essas melhorias de desempenho se traduzem diretamente aos Shaders, com a maioria rodando estavelmente acima dos 60fps, mesmo em computadores antigos de 5 ou 6 anos atrás!",
+          "screenshot": "Galeria de capturas",
+          "screenshotsubheading": "Veja o que Iris tem a oferecer",
+          "screenshot1": "Capturada por Guardian usando SEUS v11",
+          "screenshot2": "Capturada por Guardian usando Sildurs Extreme VL",
+          "screenshot3": "Capturada por FoundationGames usando Complementary",
+          "screenshot4": "Capturada por IMS usando Sildurs Vibrant Extreme VL",
+          "screenshot5": "Capturada por IMS usando BSL v8",
+          "screenshot6": "Capturada por IMS usando Complementary v4.0",
+          "screenshot7": "Capturada por IMS usando Complementary v4.0",
+          "screenshot8": "Capturada por IMS usando SEUS Renewed",
+          "screenshot9": "Capturada por IMS usando Complementary v4.0",
+          "screenshot10": "Capturada por IMS usando Complementary v4.0",
+          "screenshot11": "Capturada por kinzuu_music usando Complementary v4.0",
+          "screenshot12": "Capturada por Justsnoopy30 usando Complementary v4.0 BE",
+		  "note": "Detalhe - BE indica capturas que mostram conteúdos do mod BetterEnd de paulevs",
+          "footer1": "Sobre o Projeto Iris",
+		  "footerdesc1": "O Projeto Iris é um projeto open source criado por um time de desenvolvedores aficionados em trazer a melhor experiência possível de shaders no Minecraft. Iris é desenvolvido pela comunidade para a comunidade.",
+          "footer2": "Links úteis",
+          "curseforge": "Iris no CurseForge",
+          "modrinth": "Iris no Modrinth",
+          "github": "GitHub",
+          "discord": "Discord",
+          "patreon": "Patreon",
+          "copyright": "© Copyright 2021 IrisShaders. Todos os direitos reservados."
+        },
+        {
+          downloadiris": "Baixe agora - IrisShaders",

```

Comment:
Missing a quotation mark at the beginning

Reply:
Tnks, fixed